contracts/refund/Refunder.sol

```diff
@@ -1,47 +0,0 @@
-pragma solidity ^0.5.11;
```

Comment:
This and `AlphaJobsManager` files should be restored (for streamflow). Maybe we can remove them in another PR

Reply:
We will not be merging this into the `streamflow` branch , rather keep this as the `next` branch and eventually the `confluence` upgrade.